tests/functional/context/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from vyper import ast as vy_ast
+from vyper.context import namespace as ns
+
+
+@pytest.fixture(scope="session")
+def build_node():
+    """
+    Yields a helper function for generating a single Vyper AST node.
+    """
+
+    def _build_node(source):
+        source = f"""'I am a docstring.'\n{source}"""
```

Comment:
The docstring is required because if the first node is a string, it gets parsed as a docstring insead of a regular node in the body.  I agree it's not intuitive, I'll add a comment.